### ISSUE TITLE
GetMatchDetails API endpoint stopped to provide any result

### ISSUE THREAD

**Issue by Dmytro4** [ValveSoftware/Dota2-Gameplay#17910]
### Description

After the release of 7.36 "GetMatchDetails" does not provide any result on any match, just an empty array. Many sites like Stratz and Opendota are affected as well so new matches can not be parsed.

**https://api.steampowered.com/IDOTA2Match_570/GetMatchDetails/v1?key=#&match_id=#**

**Comment by Froctnow** [comment]
+

The endpoint https://api.steampowered.com/IDOTA2Match_570/GetMatchDetails/v1 for web API stem also doesn't work

**Comment by Dmytro4** [comment]
i would like to add that it returns 500 error for any match_id

> Status Code:
> 500 Internal Server Error

**Comment by ikonduktor** [comment]
Any decisions?

**Comment by a361570441** [comment]
same issue ,  any solutions? TOT

**Comment by daveknippers** [comment]
please come back API

**Comment by ralfizzle** [comment]
Having the same issue

**Comment by eremenko1** [comment]
up

**Comment by statdota** [comment]
Same, up

**Comment by ikonduktor** [comment]
I see that dotabuff and opendota somehow can parse finished matches. Anyone know how do they do that?

**Comment by statdota** [comment]
> I see that dotabuff and opendota somehow can parse finished matches. Anyone know how do they do that?

IDOTA2Match_570/GetMatchHistoryBySequenceNum

**Comment by dota-coach** [comment]
@Valve: Shall we all migration to GetMatchHistoryBySequenceNum?

**Comment by Noxville** [comment]
@dota-coach That means keeping up with that endpoint, which is pretty annoying to do on a single IP; and you won't get a bunch of the data (draft info, ability upgrade info, etc).

**Comment by snudel** [comment]
thats sad. did you guys migrate?

**Comment by roserens** [comment]
> @dota-coach That means keeping up with that endpoint, which is pretty annoying to do on a single IP; and you won't get a bunch of the data (draft info, ability upgrade info, etc).

Yes. You need 2 IPs for full parsing. I have 2 for loading the ca. 1 mn matches per day and then I can scale up to 8 IPs, if I need to catch-up (for whatever reason).

**Comment by roserens** [comment]
> thats sad. did you guys migrate?

I'm working on a migration to the Game Coordinator now. Which is not ideal, but probably the way to go.

If you have any other options, please let me know.

**Comment by snudel** [comment]
Did you guys manage to migrate?
I tried calling GetMatchHistoryBySequenceNum but it seems to ignore start_at_match_seq_num when handing over the matchid. or is this sequence another number than the matchid?

**Comment by eremenko1** [comment]
Up

**Comment by roserens** [comment]
Yes, I migrated to the game coordinator. But as @Noxville pointed out there is some information missing, e.g. ability upgrades.

Therefore I first fetch the data from the game coordinator and then fetch the meta data from the valve server. A little cumbersome, but it works.

I also published the meta data fetcher & parser as npm package, if you use javascript you can reuse it. It's quite lean: https://www.npmjs.com/package/dota-meta-parser

**Comment by eremenko1** [comment]
upd
